Cannot find module on 5.1.1

i am using node 11 and did a fresh installation on a new project.

I am getting Error: Cannot find module '@destinationstransfers/passkit' when i start the server.
I dont get this error if i try with 4.3.2
I have tried deleting node modules package and package-lock json several times and doing fresh install but I keep getting same error on below line
`const { Template } = require("@destinationstransfers/passkit");`

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
     "passkit-keys": "./bin/passkit-keys"
   },
   "files": [
+    "index.js",
     "src",
     "bin/passkit-keys"
   ],

